### DISCUSSION TITLE
Upload Failed/Upload aborted

### DISCUSSION THREAD

**Discussion by ofiratt** [arendst/Tasmota#14668] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/1330843/151692601-41c71ae0-977e-498b-a49f-6b770acc989d.png)
Hi guys @s-hadinger, I have "Upload Failed" Upload aborted issue

Steps:
1. solder all the wires
2. download and install https://github.com/tasmota/tasmotizer/releases/download/v.1.2/tasmotizer-1.2.exe
3. removing the ground from 100
4. reboot
5. connect to my AP
6. run in console "Backlog weblog 3; SetOption65 1; Module 75"
7. upload "https://github.com/arendst/Tasmota/blob/development/tools/fw_SonoffZigbeeBridge_ezsp/ncp-uart-nsw_6.7.9_115200.ota" around 188kb

Result: 

08:53:55.615 UPL: Progress 100 kB
08:53:57.365 UPP: Multicast (re)joined
08:53:57.366 UPL: Transfer 192240 bytes
08:53:57.367 UPL: Successful 192240 bytes
08:53:57.378 HTP: Upload transfer
08:53:57.450 XMD: Init bootloader
08:53:57.451 ZIG: Bootloader pin 5
08:54:00.669 XMD: Init sync
08:54:30.672 XMD: SYNC timeout
08:54:30.673 XMD: Failed
08:54:30.673 ZIG: Bootloader pin 5
08:54:31.535 HTP: Upload done

and I get "Upload Failed" in the Firmware Upgrade page

** I reaped this process by installing the tasmota-zbbridge.bin and tried also from the release and I tried the firmware version 6.7.9 & 6.7.8 & 6.7.6

when I run "module" in the console I get:

09:31:15.015 CMD: module
09:31:15.016 SRC: WebConsole from xxx.xxx.xxx.xxx
09:31:15.017 CMD: Grp 0, Cmnd 'MODULE', Idx 1, Len 0, Data ''
09:31:15.020 RSL: RESULT = {"Module":{"75":"Sonoff ZbBridge"}}

Please Help

**Comment by Jason2866** [comment, marked as answer]
Nearly all cases we heard of this problems where hardware issues. Traces rx / tx broken from esp to zigbee module. Check the connections with a multimeter.

**Reply by ofiratt** [reply]
I don't understand as I manage to install Tasmota on the device, my issue is at the OTA phase

**Reply by barbudor** [reply]
We have seen a lot of case where people, when soldering/unsoldering wires to flash Tasmota are damaging the serial traces on the PCB between the ESP and the zigbee chip. If the serial link is damaged, you can't neither upload a new firmware nor even communicate with teh Zigbee chip.

**Reply by Jason2866** [reply]
The zigbee firmware is uploaded via rx / tx (internally) to the zigbee module.

**Reply by ofiratt** [reply]
ohh, so you mean that my module is faulty...
Interesting, I'll try to look up close to the soldering to see what can I find

**Reply by nebbiadigiorno** [reply]
what i have to test with te multimeter?

**Reply by sfromis** [reply]
If your wiring is making the right connections, or "accidents" happened while trying to do it...

**Reply by fflores97** [reply]
Came here to say this was the case for me. I didn't have a proper soldering station when I tried flashing for the first time and so I burned the trace from the ESP to the zigbee module. Later soldered a jumper wire to the resistor right before the zigbee module and it flashed upgrade without any issues
